sections/06_discussion.md

```diff
@@ -210,6 +210,196 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
+to a common stimuli. Another application used DBNs for joint unsupervised
+feature learning from cancer datasets containing gene expression,
+DNA methylation, and miRNA expression data [@tag:Liang2015_exprs_cancer].
+This approach allowed for the capture of intrinsic relationships in
+different modalities and for better clustering performance over conventional
+k-means based methods.
+Multimodal learning with CNNs is usually implemented as a collection of
+individual networks in which each learns representations from single data type.
+These individual representations are further concatenated before or within
+fully-connected layers. FIDDLE [@tag:Eser2016_fiddle] is an example of
+multimodal CNNs that represents an ensemble of individual networks that take
+as inputs a number of genomic datasets, including NET-seq, MNase-seq, ChIP-seq,
+RNA-seq, and raw DNA sequence to predict Transcription Start Site-seq
+(TSS-seq) outputs. The combined model radically improves performance over
+separately trained datatype-specific networks, suggesting that it learns the
+synergistic relationship between datasets.
+
+Multi-task learning (MTL) is an approach related to transfer learning.
+In an MTL framework a model co-learns a number of tasks simultaneously such that
+features are shared across them.
+DeepSEA framework [@tag:Zhou2015_deep_sea] implemented a multi-task joint
+learning of diverse chromatin factors sharing predictive features from raw
+DNA sequence. This allowed, for example, a sequence feature that is effective
+in recognizing binding of a specific TF to be simultaneously used by
+another predictor for a physically interacting TF.
+Similarly, TFImpute [@tag:Qin2017_onehot], a CNN-RNN architecture learned
+information shared across transcription factors and cell lines to predict
+cell-specific TF binding for TF-cell line combinations based on only a
+small fraction (4%) of the combinations using available ChIP-seq data. On
+multiple test sets that excluded specific TFs and cell lines, TFImpute showed
+comparable or superior performance compared to the state-of-the-art.
+Yoon et al.[@tag:Yoon2016_cancer_reports], previously discussed in the section
+on Electronic Health Records, demonstrated that predicting the primary cancer
+site from the cancer pathology reports together with its laterality
+substantially improved the performance for the latter task, suggesting that
+MTL can effectively leverage the commonality between two tasks using
+a shared representation.
+A number of studies previously mentioned in the section on developing new
+treatments employed multi-task learning approach to predict a large number
+of compound and target interactions for drug discovery
+[@tag:Dahl2014_multi_qsar @tag:Ramsundar2015_multitask_drug]
+and drug toxicity prediction
+[@tag:Mayr2016_deep_tox @tag:Hughes2016_macromol_react]. Kearnes et al.
+[@tag:Kearnes2016_admet] did a systematic comparison of single-task and
```

Comment:
The Kearnes conclusions were somewhat more complex that this.  They found that adding large numbers of less-related tasks actually hurt performance sometimes, right?  It could be preferable to do MTL with a smaller number of more similar tasks.

Some of this is in Treat, so we could shorten the Kearnes discussion here.